### PR TITLE
Provide Endorsed Evidence from Launcher

### PR DESCRIPTION
**Pull request by ipetr0v**

### CODE REVIEW

File: java/src/main/java/com/google/oak/transport/GrpcStreamingTransport.java [hunk 1]
```diff
@@ -16,11 +16,6 @@
 
 package com.google.oak.transport;
 
-import java.time.Duration;
```
Comment (conradgrobler) [inline java/src/main/java/com/google/oak/transport/GrpcStreamingTransport.java:19]: I don't think these formatting changes should be part of the PR.

File: oak_containers_kernel/configs/6.1.33/minimal.config [hunk 2]
```diff
@@ -11,7 +11,6 @@ CONFIG_AS_VERSION=24000
 CONFIG_LD_IS_BFD=y
 CONFIG_LD_VERSION=24000
 CONFIG_LLD_VERSION=0
-CONFIG_RUST_IS_AVAILABLE=y
```
Comment (conradgrobler) [inline oak_containers_kernel/configs/6.1.33/minimal.config:14]: Why is this change needed?

File: oak_containers_launcher/src/lib.rs [hunk 3]
```diff
@@ -81,6 +100,12 @@ pub struct Launcher {
     vmm: qemu::Qemu,
     server: JoinHandle<Result<(), anyhow::Error>>,
     host_proxy_port: u16,
+    attestation_endorsement: AttestationEndorsement,
+    // Endorsed Attestation Evidence consists of Attestation Evidence (initialized by the
+    // Orchestrator) and Attestation Endorsement (initialized by the Launcher).
+    endorsed_attestation_evidence: Option<AttestationBundle>,
+    // Reciever that is used to get the Attestation Evidence from the server implementation.
```
Comment (conradgrobler) [inline oak_containers_launcher/src/lib.rs:107]: nit: `Receiver`

File: oak_containers_launcher/src/lib.rs [hunk 4]
```diff
@@ -153,6 +195,32 @@ impl Launcher {
...
     }
 
+    /// Gets the endorsed attestation evidence that the untrusted application can send to remote
+    /// clients, which will verify it before connecting.
+    pub async fn get_endorsed_evidence(&mut self) -> anyhow::Result<AttestationBundle> {
+        // If we haven't received an attestation evidence, wait for it.
+        if let Some(receiver) = self.attestation_evidence_receiver.take() {
+            // Set a timeout of 5 minutes, since we don't want to wait forever if the VM didn't
+            // start properly.
+            let sleep = tokio::time::sleep(tokio::time::Duration::from_secs(300));
```
Comment (conradgrobler) [inline oak_containers_launcher/src/lib.rs:205]: Can be cdone as future cleanup, but since we follow the same pattern here and with the ready notification we should think of a way to reduce the code duplication. For a start the timeout should be a constant, but perhaps we can also create a wrapper to wait with timeout (or perhaps tokio already has one).

**Review by conradgrobler — APPROVED** [review]

File: oak_containers_kernel/configs/6.1.33/minimal.config (hunk 2 above)
Comment (ipetr0v) [reply oak_containers_kernel/configs/6.1.33/minimal.config:14]: I don't know how it appeared. Probably after formatting. Will try putting it back.

File: java/src/main/java/com/google/oak/transport/GrpcStreamingTransport.java (hunk 1 above)
Comment (ipetr0v) [reply java/src/main/java/com/google/oak/transport/GrpcStreamingTransport.java:19]: It happened automatically. Will revert.

File: oak_containers_launcher/src/lib.rs (hunk 4 above)
Comment (ipetr0v) [reply oak_containers_launcher/src/lib.rs]: Added a constant

File: oak_containers_launcher/src/lib.rs (hunk 3 above)
Comment (ipetr0v) [reply oak_containers_launcher/src/lib.rs]: Done

File: oak_containers_kernel/configs/6.1.33/minimal.config (hunk 2 above)
Comment (ipetr0v) [reply oak_containers_kernel/configs/6.1.33/minimal.config]: Done

File: java/src/main/java/com/google/oak/transport/GrpcStreamingTransport.java (hunk 1 above)
Comment (ipetr0v) [reply java/src/main/java/com/google/oak/transport/GrpcStreamingTransport.java]: Done
